packages/core/src/Stores/ui-store.js

```diff
@@ -25,6 +25,10 @@ export default class UIStore extends BaseStore {
     @observable is_reports_visible          = false;
     @observable is_cashier_visible          = false;
 
+    // Stop loss & Deal cancellation checkbox
+    @observable should_show_stop_loss_warning         = true;
+    @observable should_show_deal_cancellation_warning = true;
```

Comment:
Can these states not be managed locally? So we can avoid adding it to the common UI store.

Reply:
Unfortunately no cos we needed to store these 2 values in store so it doesn't get reset on page refresh